client/state/stats/lists/reducer.js

```diff
@@ -84,6 +85,27 @@ export const items = withSchemaValidation( itemSchema, ( state = {}, action ) =>
 					},
 				},
 			};
+		}
+
+		case ALL_SITES_STATS_RECEIVE: {
+			const { query, data } = action;
+			const newState = { ...state };
+
+			for ( const siteId in data.stats ) {
+				const siteData = data.stats[ siteId ];
+				const statType = 'statsSummary'; // ??
+				const queryKey = getSerializedStatsQuery( query );
```

Comment:
Seems like the `queryKey` constant can be declared outside of the `for` loop as well.